beta-src/src/interfaces/state/MemberData.ts

```diff
@@ -0,0 +1,19 @@
+import IntegerRange from "../../types/IntegerRange";
+
+export interface MemberData {
+  bet: IntegerRange<5, 96>;
+  country: string;
+  countryID: number;
+  excusedMissedTurns: IntegerRange<0, 5>;
+  missedPhases: number;
+  newMessagesFrom: number[];
+  online: boolean;
+  orderStatus: any;
```

Comment:
booooooo will unknown work?

Reply:
Probably, is unknown generally better than any? Does it infer the first type it encounters if it is unknown?

Reply:
good point...i guess the real question is why can't we type it more safely?